src/js/site/main.js

```diff
@@ -260,6 +260,7 @@ $(document).ready(function () {
 
   // Give article headings direct links to anchors
   $('article h1, article h2, article h3, article h4, article h5, article h6')
+    .not('.card')
```

Comment:
need to change to '.card article h1, article h2, article h3, article h4, article h5, article h6'

Reply:
and add to correct place